src/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -249,6 +264,19 @@ export default function CurrencyList({
           <ImportRow style={style} token={token} showImportView={showImportView} setImportToken={setImportToken} dim />
         )
       } else if (currency) {
+        const eventProperties = {
+          token_symbol: token?.symbol,
+          token_address: token?.address,
+          is_suggested_token: false,
+          is_selected_from_list: true,
+          scroll_position: '',
+          token_list_index: index + 1, // 1-indexed
```

Comment:
is there a reason to 1-index? as an engineer i always default to 0 index

Reply:
hmm for PM's and research folks using this data :P i'm kind of indifferent though. let's see what Will thinks

Reply:
true, changed to 0 indexed, Will agreed.